app/scripts/lib/auth-errors.js

```diff
@@ -198,6 +198,14 @@ define(function (require, exports, module) {
       errno: 151,
       message: t('Failed to send email')
     },
+    INVALID_TOKEN_VERIFICATION_CODE: {
+      errno: 152,
+      message: t('Invalid code')
```

Comment:
Can we use a more descriptive message here? Would we confuse if someone tell us " I got an invalid code error", we have so many codes that can be invalid